src/ChannelServer/World/MonsterBuilder.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Melia.Shared.Const;
+using Melia.Shared.World;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Melia.Channel.World
+{
+    class MonsterBuilder
```

Comment:
Builders are easier to use. The way it's done now need from you to specify all major values manually to object, but builders will done it by themselfs with better visualization.